internal/local/structure_v2.go

```diff
@@ -272,8 +274,9 @@ func parseEndpointsV2(rootDir string) (EndpointStructure, error) {
 	return EndpointStructure{configs}, nil
 }
 
-func parseDataSources(rootDir string) ([]DataSourceStructure, error) {
+func parseDataSources(rootDir string) ([]DataSourceStructure, []map[string]interface{}, error) {
 	var out []DataSourceStructure
```

Comment:
[nit] now that this function returns two values, could this be changed to `dataSources` or something describing the object?